### DISCUSSION TITLE
UX when clicking on a profile

### DISCUSSION THREAD

**Discussion by Geobert** [cheeaun/phanpy#564] [Ideas]
When we click on an author, we have a profil card only. Then if we want to see the user’s toots, we click again and we are served the same profile card followed by the posts. 

Wouldn’t it be better to have directly the second click result as it contains the first operation infos anyway?

**Comment by cheeaun** [comment]
Do you mean the second click to show the posts of the profile directly instead of the profile card?

**Reply by Fastidious** [reply]
I believe that’s what OP meant, yes.

**Reply by Geobert** [reply]
I meant directly go to posts yes :)

**Reply by cheeaun** [reply]
@Geobert is this mostly on mobile or desktop?

**Comment by Geobert** [comment]
It is for both :)

--
Geobert Quach

Envoyé depuis Proton Mail pour Android

-------- Message d'origine --------

**Comment by wezm** [comment]
I came to the repo looking to report this too. I use Phanpy on desktop only. When I click on a user's profile I almost always want to see their other posts and as OP mentioned this I think requires two clicks: one to open the pop-over, and another to open the profile page proper. I think in an ideal world clicking a profile would open it the side-bar, the same as what happens when clicking a post.
